src/components/Clock/Clock.css

```diff
@@ -0,0 +1,49 @@
+@import url('https://fonts.googleapis.com/css?family=Lato');
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
```

Comment:
Er du sikker på at du ønsker å endre tekst på body? Det vil si hele siden. Litt usikker på hvordan det funker når du kun importerer det til clock. Går kanskje fint?

Reply:
Okei, greit å vite. Nei, hadde ikke tenkt til å endre på hele siden. Tenkte at det var "body" for akkurat den komponenten som bruker den css'en. Skal se på det!